### PR TITLE
fix/108305 tags event categories block

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/108305

### CODE REVIEW

File: src/modules/elements/terms-list/element.js [hunk 1]
```diff
@@ -46,12 +47,15 @@ class TaxonomiesElement extends Component {
 		return terms.filter( term => term.parent === parentId );
 	}
 
-	getTermListClassName( level ) {
-		const { className } = this.props;
-		return `tribe-editor__terms__list tribe-terms__list-level-${ level } ${ className }`;
-	}
+	getTermListClassName = level => (
```
Comment (mitogh) [inline src/modules/elements/terms-list/element.js]: Just for consistency reasons we should keep using the parenthesis around the param even if is only one.  `( level )`

File: src/modules/elements/terms-list/element.js [hunk 2]
```diff
@@ -46,12 +47,15 @@ class TaxonomiesElement extends Component {
...
+		`tribe-editor__terms__list tribe-editor__terms__list--level-${ level }`
+	);
+
+	getTermListItemClassName = level => (
```
Comment (mitogh) [inline src/modules/elements/terms-list/element.js]: Same as above.

File: src/modules/elements/terms-list/element.js (hunk 1 above)
Comment (paulmskim) [reply src/modules/elements/terms-list/element.js]: what linting rules are we using for ES6 here? I'm used to Airbnb's linting rules, and this is how they do it, but if we're doing something different, I'm down for that too.

Comment (mitogh) [reply src/modules/elements/terms-list/element.js]: I think is a set of custom ones + WP Calypso.

- https://github.com/moderntribe/eslint-config-products/

Comment (paulmskim) [reply src/modules/elements/terms-list/element.js]: I actually had a look at our eslint rules and, unless `products/react` has others specified, if we are using the default (`all`) for the `no-extra-parens` rule, the default for single parameter with `returnAssign` is as what is seen here: https://eslint.org/docs/rules/no-extra-parens

Comment (paulmskim) [reply src/modules/elements/terms-list/element.js]: Sorry, disregard my last one. The `arrow-parens` rule on `always` (default) has parentheses around the parameters in all cases: https://eslint.org/docs/rules/arrow-parens

i'll make the changes.

**Pull request merged**